File: Java/Card.java
import java.util.Date;

public class Card extends Payment
{
    Integer cardNumber;
    Date date;
    Integer cvv;

    public Card(Integer id, Integer cardNumber, Date date, Integer cvv)
    {
        super(id);
        this.cardNumber = cardNumber;
        this.date = date;
        this.cvv = cvv;
    }
}

File: PHP/Card.php
<?php
require_once("Payment.php");

class Card extends Payment
{
    public $cardNumber;
    public $date;
    public $cvv;

    public function _construct($id, $cardNumber, $date, $cvv)
    {
        parent::_construct($id);
        $this->$cardNumber = $cardNumber;
        $this->$date = $date
        $this->$cvv = $cvv;
    }
}

?>